Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingController.swift

```diff
@@ -19,7 +19,7 @@ open class EpoxySwiftUIHostingController<Content: View>: UIHostingController<Con
 
   /// Creates a `UIHostingController` that optionally ignores the `safeAreaInsets` when laying out
   /// its contained `RootView`.
-  public convenience init(rootView: Content, ignoreSafeArea: Bool) {
+  public convenience init(rootView: Content, ignoreSafeArea: Bool, ignoreKeyboardAvoidance: Bool) {
```

Comment:
just realized both should be `ignores` for consistency

Reply:
If we rename the API, it'll be a breaking change. That's fine but just something to keep in mind since it will change the `semver` version when we tag another release off of master.